cluster-autoscaler/core/scaleup/orchestrator/orchestrator.go

```diff
@@ -240,35 +230,8 @@ func (o *ScaleUpOrchestrator) ScaleUp(
 		}
 	}
 
-	// Recompute similar node groups in case they need to be updated
-	bestOption.SimilarNodeGroups = o.ComputeSimilarNodeGroups(bestOption.NodeGroup, nodeInfos, schedulablePodGroups, now)
-	if bestOption.SimilarNodeGroups != nil {
-		// if similar node groups are found, log about them
-		similarNodeGroupIds := make([]string, 0)
-		for _, sng := range bestOption.SimilarNodeGroups {
-			similarNodeGroupIds = append(similarNodeGroupIds, sng.Id())
-		}
-		klog.V(2).Infof("Found %d similar node groups: %v", len(bestOption.SimilarNodeGroups), similarNodeGroupIds)
-	} else if o.autoscalingContext.BalanceSimilarNodeGroups {
-		// if no similar node groups are found and the flag is enabled, log about it
-		klog.V(2).Info("No similar node groups found")
-	}
-
-	// Balance between similar node groups.
-	targetNodeGroups := []cloudprovider.NodeGroup{bestOption.NodeGroup}
-	for _, ng := range bestOption.SimilarNodeGroups {
-		targetNodeGroups = append(targetNodeGroups, ng)
-	}
-
-	if len(targetNodeGroups) > 1 {
-		var names []string
-		for _, ng := range targetNodeGroups {
-			names = append(names, ng.Id())
-		}
-		klog.V(1).Infof("Splitting scale-up between %v similar node groups: {%v}", len(targetNodeGroups), strings.Join(names, ", "))
-	}
-
-	scaleUpInfos, aErr := o.processors.NodeGroupSetProcessor.BalanceScaleUpBetweenGroups(o.autoscalingContext, targetNodeGroups, newNodes)
+	// Balance scale ups between similar node groups.
```

Comment:
Since you're extracting this logic to a new function, I'd expect that function name to be self-explanatory enough not to require an additional comment about what the function will do. So, I'd drop the comment and - if you think it would improve readability - make the function name even more self-descriptive.

Reply:
Totally agree, changed. I just wanted to align to other comments in this function :)